### PR TITLE
PLANET-5761 Remove jQuery from search

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [PLANET-5761](https://jira.greenpeace.org/browse/PLANET-5761)

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [sinope](https://www-dev.greenpeace.org/test-sinope) | [admin](https://www-dev.greenpeace.org/test-sinope/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-sinope/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-sinope) | [composer-local.json](https://github.com/greenpeace/planet4-test-sinope/blob/main/composer-local.json)

:watch: 2021.12.24 13:18:03

File: templates/tease-search.twig [hunk 1]
```diff
@@ -25,7 +27,9 @@
 			alt="{{ fn('esc_attr', post.thumbnail_alt|default( post.post_title ) )|raw }}"
 			data-ga-category="Search Results"
 			data-ga-action="Image"
-			data-ga-label="{{ ga_page_type }}" />
+			data-ga-label="{{ ga_page_type }}"
+			onclick="window.location.href='{{ post_link }}'"
```
Comment (Inwerpsel) [inline templates/tease-search.twig:31]: If we want the image to be a clickable link, we should use an `a` tag around it.

Implementing our own link with JS misses a lot of things about the UX of links:
- focus states are different
- not accessible
- open in a new tab won't work
- probably more

**Comment by Inwerpsel** [comment]
@mleray we currently have a ticket to solve the issue of duplicate results @lithrel is working on. That will likely involve making search use the Ajax calls exclusively, so also for the first page of results.

I suppose we should do both at the same time to avoid converting code that will be removed/significantly altered.

**Comment by lithrel** [comment]
> That will likely involve making search use the Ajax calls exclusively

@Inwerpsel  Still working on this topic and not sure I will do that, I still think it's better to understand the difference between the 2 search requests and properly unify the query behind them; adding an ajax call is just adding a request for the client, I don't really see a benefit to this at the moment

**Comment by Inwerpsel** [comment]
> > That will likely involve making search use the Ajax calls exclusively
> 
> @Inwerpsel Still working on this topic and not sure I will do that, I still think it's better to understand the difference between the 2 search requests and properly unify the query behind them; adding an ajax call is just adding a request for the client, I don't really see a benefit to this at the moment

Good point, we should definitely sort out what the query is supposed to do. Using the Ajax calls seemed like the quickest way to get there and we would be able to already drop much of the complexity of search.

It has a few more benefits. We could likely cache the initial (empty) search HTML in Cloudflare. So the total time would not be significantly more, and perhaps even less because the initial call can be made more efficient than the current request that creates the first page HTML.

That being said, it would also need to have the sidebar filters, which may turn out more complicated (I guess it would need to be added to the first page data). So we can indeed still consider fixing the separate query.

**Pull request closed without merging**